### PR TITLE
Change default ttlaction from delete to free

### LINKED ISSUE #12316: can alluxio change default ttlaction from delete to free?
**Alluxio Version:
2.3

it is very dangerous for ttlaction default to be delete. ufs file may be deleted

### PR DESCRIPTION
**Pull request by maobaolong**
Fix #12316

### CODE REVIEW

File: tests/src/test/java/alluxio/client/hadoop/FileSystemRenameIntegrationTest.java [hunk 1]
```diff
@@ -236,7 +236,7 @@ public void basicRenameTest7() throws Exception {
     // Due to Hadoop 1 support we stick with the deprecated version. If we drop support for it
     // FSDataOutputStream.hflush will be the new one.
     //#ifdef HADOOP1
-    o.sync();
+//    o.sync();
```
Comment (apc999) [inline tests/src/test/java/alluxio/client/hadoop/FileSystemRenameIntegrationTest.java:239]: @maobaolong  is this change required for this PR?

Comment (maobaolong) [reply tests/src/test/java/alluxio/client/hadoop/FileSystemRenameIntegrationTest.java]: Sorry for this.

**Comment by maobaolong** [comment]
@apc999 Thanks for your review and point out the unrelated change line, I've reverted it and push a new commit, PTAL.

**Comment by apc999** [comment]
@maobaolong I had a discussion with @calvinjia  , and we will look for a more complete solution on this as discussed https://github.com/Alluxio/alluxio/issues/12316.
I will close this PR temporarily for now (may reopen it later)
